### PR TITLE
Add Mono 4.6.2.16

### PR DESCRIPTION
**Pull request by dominikhahn**

### CODE REVIEW

**Comment by akoeplinger** [comment]
Thank you!

File: 4.6.2.16/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM debian:wheezy
+
+MAINTAINER Jo Shields <jo.shields@xamarin.com>
+
+#based on dockerfile by Michael Friis <friism@gmail.com>
+
+ENV MONO_VERSION 4.6.2.16
+ENV DEBIAN_FRONTEND noninteractive
```
Comment (akoeplinger) [inline 4.6.2.16/Dockerfile]: @dominikhahn where did you get this from? It's not recommended by Docker and I had to remove it.

Comment (dominikhahn) [inline 4.6.2.16/Dockerfile]: So sorry about that one! Tried something and forgot to remove it.

Comment (akoeplinger) [inline 4.6.2.16/Dockerfile]: No worries, I almost made a similar mistake a while ago when copying the files for a new release :smile:

We should probably start automating this.

**Pull request merged**